css/amp.css

```diff
@@ -759,3 +759,19 @@ amp-accordion > section > :last-child {
 amp-accordion > section[expanded] > :last-child {
   display: block !important;
 }
+
+/**
+ * amp-story
+ */
+ amp-story, amp-story-page {
+  display: block;
```

Comment:
Static positioning OK? Not relative or absolute? Or is this up to the actual extension `amp-story.css`?

Reply:
Static positioning is okay here.  Since the height is `100vh` and the pages will lay out vertically, we'll start preloading resources for the first few viewports (based on whatever the runtime does by default).  When `amp-story.css` is loaded, it will override this to style the story appropriately.